server/routers/restaurants.test.js

```diff
@@ -16,61 +20,81 @@ describe('restaurants router', () => {
     app.use('/restaurants', restaurantsRouter);
   });
```

Comment:
how does this work when app is only defined on line 16, and is not assigned to express etc?

Reply:
oh so `beforeAll()` runs before any of your tests, so any setup you want can be done there, and then `afterAll()` can do any cleanup you need.  The let is done on line 16 so that every `test()` can refer to app.  Also, not sure if GitHub condensed the code, but `app = express()` is right above the `app.use()` line.